documentation/main_multiprocess_example.py

```diff
@@ -47,83 +52,81 @@ def main() -> int:
         ADD_RANDOM_TO_CONCATENATOR_QUEUE_MAX_SIZE,
     )
 
-    # Prepare processes
-    # Data path: countup_worker to add_random_worker to concatenator_workers
-    # Play with these numbers to see process bottlenecks
-    countup_workers = [
-        mp.Process(
-            target=countup_worker.countup_worker,
-            args=(
-                3,
-                100,
-                countup_to_add_random_queue,
-                controller,
-            ),
-        ),
-        mp.Process(
-            target=countup_worker.countup_worker,
-            args=(
-                2,
-                200,
-                countup_to_add_random_queue,
-                controller,
-            ),
-        ),
-    ]
-    countup_manager = worker_manager.WorkerManager(countup_workers)
-
-    add_random_workers = [
-        mp.Process(
-            target=add_random_worker.add_random_worker,
-            args=(
-                252,
-                10,
-                5,
-                countup_to_add_random_queue,
-                add_random_to_concatenator_queue,
-                controller,
-            ),
+    # Worker arguments
+    result, countup_worker_args = worker_manager.WorkerProperties.create(
```

Comment:
Rename variable to `countup_worker_properties` . Do similar everywhere else (this file and main 2024).